src/commands/scanner/run.ts

```diff
@@ -174,7 +176,10 @@ export default class Run extends ScannerCommand {
 			// Default properly, again, as we did earlier.
 			const format: OUTPUT_FORMAT = this.flags.format || OUTPUT_FORMAT.TABLE;
 			// If we're just supposed to dump the output to the console, what precisely we do depends on the format.
-			if (format === OUTPUT_FORMAT.CSV && typeof output === 'string') {
+			if (format === OUTPUT_FORMAT.JSON && typeof output === 'string') {
+				// JSON is also just one giant string that we can dump directly to the console.
```

Comment:
This is probably the pettiest nit I've ever picked, but the 'also' should move to line 183.

Reply:
It shows you care, @jfeingold35